app/javascript/components/public_health/CustomExport.js

```diff
@@ -272,7 +272,7 @@ class CustomExport extends React.Component {
                                     : filter.relativeOption}
                                 </span>
                               )}
-                              {filter.filterOption?.type === 'multi' && (
+                              {filter.filterOption?.type === 'combination' && (
```

Comment:
we still need to support the `multi` type so:
```suggestion
                              {['combination', 'multi'].includes(filter.filtlerOption?.type) && (
```

Reply:
The `multi` type is rendered differently than `combination`, so I'll work on this!

Reply:
Done!

Reply:
Removed `;`

Reply:
I'm not seeing any of the values showing up now

![Screen Shot 2021-08-10 at 1 43 49 AM](https://user-images.githubusercontent.com/9956561/128814421-6ae29cf1-0f2a-4747-ad9d-a0b6c9d85ea1.png)

Reply:
hmm that's weird, I am. Did you select values in the multi-select? I'll look into this and get back to you!

<img width="421" alt="Screen Shot 2021-08-10 at 08 17 50" src="https://user-images.githubusercontent.com/55925276/128865411-0323b8fe-739a-4429-9b4f-19d9a810bfa0.png">

Reply:
this seems like it might be an intermittent thing, I am seeing the options show up properly most of the time now, but not confident enough to say that it's not a problem